README.md

```diff
@@ -1,33 +1,27 @@
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
-## Hyperledger Fabric Samples
-
-Please visit the [installation instructions](http://hyperledger-fabric.readthedocs.io/en/latest/install.html)
-to ensure you have the correct prerequisites installed. Please use the
-version of the documentation that matches the version of the software you
-intend to use to ensure alignment.
-
-## Download Binaries and Docker Images
-
-The installation instructions will utilize `scripts/bootstrap.sh` (available in the fabric repository)
-script to download all of the requisite Hyperledger Fabric binaries and docker
-images, and tag the images with the 'latest' tag. Optionally,
-specify a version for fabric, fabric-ca and thirdparty images. If versions
-are not passed, the latest available versions will be downloaded.
-
-The script will also clone fabric-samples repository using the version tag that
-is aligned with the Fabric version.
-
-You can also download the script and execute locally:
-
-```bash
-# Fetch bootstrap.sh from fabric repository using
-curl -sS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh -o ./scripts/bootstrap.sh
-# Change file mode to executable
-chmod +x ./scripts/bootstrap.sh
-# Download binaries and docker images
-./scripts/bootstrap.sh [version] [ca version] [thirdparty_version]
-```
+# Hyperledger Fabric Samples
+
+You can use Fabric samples to get started working with Hyperledger Fabric, explore important Fabric features, and learn how to build applications that can interact blockchain networks using the Fabric SDK's. To learn more about Hyperledger Fabric, visit the [Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/master).
+
+## Getting started with the Fabric samples
+
+To use the Fabric samples, you need to download the Fabric Docker images and the Fabric CLI tools. First, make sure that you have installed all of the [Fabric prerequisites](https://hyperledger-fabric.readthedocs.io/en/master/prereqs.html). You can then follow the instructions to [Install the Fabric Samples, Binaries, and Docker Images](https://hyperledger-fabric.readthedocs.io/en/master/install.html) in the Fabric documentation. In addition to downloading the Fabric images and tool binaries, the instructions will clone the Fabric samples on your local machine.
```

Comment:
the instructions will clone -> the instructions will make you clone

Reply:
done